macos/FluentUI/Separator.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+@objc(MSFSeparatorOrientation)
+public enum SeparatorOrientation: Int {
+    case horizontal
+    case vertical
+}
+
+@objc(MSFSeparator)
+open class Separator: NSView {
+	let orientation: SeparatorOrientation
+	
+	/// Initializes a separator in the specified orientation
+	/// - Parameter orientation: The orientation of the separator, vertical or horizontal
+	@objc public init(orientation: SeparatorOrientation) {
+		self.orientation = orientation
+		super.init(frame: .zero)
+		wantsLayer = true
+		if let layer = layer {
+			layer.backgroundColor = separatorColor.cgColor
+		}
+	}
+
+	@available(*, unavailable)
+	required public init?(coder aDecoder: NSCoder) {
+		preconditionFailure()
+	}
+	
+	open override var wantsUpdateLayer: Bool {
+		return true
+	}
+
+	open override func updateLayer() {
+		if let layer = layer {
+			layer.backgroundColor = separatorColor.cgColor
+		}
+	}
+	
+	open override var intrinsicContentSize: CGSize {
+		switch orientation {
+		case .horizontal:
+			return CGSize(width: NSView.noIntrinsicMetric, height: separatorThickness)
+		case .vertical:
+			return CGSize(width: separatorThickness, height: NSView.noIntrinsicMetric)
+		}
+	}
+	
+	private var separatorColor: NSColor {
```

Comment:
let instead of var possible? Or no

Reply:
`'let' declarations cannot be computed properties`

Nope :(

Reply:
private let separatorColor: NSColor = {
       // code
    }()

---
In reply to: [446368706](https://github.com/microsoft/fluentui-apple/pull/103#discussion_r446368706) [](ancestors = 446368706)